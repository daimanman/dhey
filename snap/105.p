{"C":1,"Duration":7583712536,"EndTime":1547003217391103000,"Err":"","FileId":"1","InputFileName":"","Method":"GET","N":10,"P":null,"Payload":"","QH":{},"Remark":"1222 test pg ","ReqNums":10,"StartTime":1547003209807390500,"Status":1,"TaskId":105,"Type":"FIXED","Url":"https://www.cwbaoguowang.com/images/open.png","Z":""}