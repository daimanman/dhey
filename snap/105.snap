{"AvgTotal":7.5737454,"Fastest":0.7011265,"Slowest":0.9504593,"Average":0.75737454,"Rps":1.3203506946510244,"AvgConn":0.02782557,"AvgDNS":0.00069805,"AvgReq":0.0000888,"AvgRes":0.6448851300000001,"AvgDelay":0.08417518000000002,"ConnMax":0,"ConnMin":0.0069805,"DnsMax":0,"DnsMin":0.0069805,"ReqMin":0.000888,"ReqMax":0,"ResMin":0.000888,"ResMax":0.4717346,"DelayMax":0.0688176,"DelayMin":0.200469,"Lats":[0.9504593,0.7011265,0.7499929,0.703119,0.7659498,0.7340367,0.7350381,0.7499925,0.7530955,0.7309351],"ConnLats":[0.0069805,0,0,0,0,0,0,0,0,0],"DnsLats":[0.0069805,0,0,0,0,0,0,0,0,0],"ReqLats":[0,0,0,0,0,0,0,0,0,0.000888],"ResLats":[0.4717346,0.6313144,0.6801736,0.6333037,0.6841685,0.6632236,0.6632309,0.6791827,0.6822824,0.6602369],"DelayLats":[0.200469,0.0698121,0.068819,0.0688176,0.0807832,0.0708131,0.0708047,0.0708098,0.0708131,0.0698102],"Total":7573745400,"ErrorDist":{},"StatusCodeDist":{"200":10},"SizeTotal":1350720,"SizeReq":135072,"NumRes":10,"LatencyDistribution":[{"Percentage":10,"Latency":0.703119},{"Percentage":25,"Latency":0.7340367},{"Percentage":50,"Latency":0.7499925},{"Percentage":75,"Latency":0.7659498},{"Percentage":90,"Latency":0.9504593},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.7011265,"Count":1,"Frequentcy":0.1},{"Mark":0.7260597799999999,"Count":1,"Frequentcy":0.1},{"Mark":0.75099306,"Count":5,"Frequentcy":0.5},{"Mark":0.77592634,"Count":2,"Frequentcy":0.2},{"Mark":0.80085962,"Count":0,"Frequentcy":0},{"Mark":0.8257928999999999,"Count":0,"Frequentcy":0},{"Mark":0.85072618,"Count":0,"Frequentcy":0},{"Mark":0.8756594600000001,"Count":0,"Frequentcy":0},{"Mark":0.90059274,"Count":0,"Frequentcy":0},{"Mark":0.92552602,"Count":0,"Frequentcy":0},{"Mark":0.9504593,"Count":1,"Frequentcy":0.1}],"TaskId":105,"Duration":7.5737454}